src/components/Popover/Popover.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import useDisclosure from '../../utils/useDisclosure';
+
+export interface PopoverProps {
+  /**
+   * Whether the popover should be initially open
+   */
+  isOpen?: boolean;
+
+  /**
+   * The id of the popover. Helps setting up aria controls between popover content & trigger
+   */
+  popoverId?: string;
+}
+
+interface PopoverContext {
+  popoverId?: string;
```

Comment:
Semantically the `popoverId` is vital so we can use [auto-id](https://reach.tech/auto-id) as a default value.

Reply:
I've removed custom overriding since it's 1 less thing to worry about 😅

Reply:
👍